app/src/main/java/com/cornellappdev/transit/models/Place.kt

```diff
@@ -12,24 +13,22 @@ data class QueryResult(
     @Json(name = "busStops") var stops : List<Place>
 )
 
-//TODO: Deprecated, start migrating to Place data class
 /**
- * DEPRECATED: Data class representing a TCAT stop
+ * Enum class representing the types of TCAT places
  */
-data class Stop(
-    @Json(name = "lat") var latitude: Double,
-    @Json(name = "long") var longitude: Double,
-    @Json(name = "name") var name: String,
-    @Json(name = "type") var type: String,
-)
+enum class Type() {
```

Comment:
Any reason for `()` here?

Reply:
Also, the name `Type` is a little generic. Maybe something like `PlaceType`?

Reply:
Lol these were changes on my branch so I'll fix them.

Reply:
no reason for `()` and will change name to `PlaceType`.

Reply:
No worries @Mihilih ill fix them on this branch lol